<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Читалка комиксов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelAuthor">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>16</y>
      <width>57</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Автор:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAuthorValue">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>16</y>
      <width>157</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>16</y>
      <width>67</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameValue">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>16</y>
      <width>147</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPublisher">
    <property name="geometry">
     <rect>
      <x>754</x>
      <y>16</y>
      <width>67</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Издатель:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPublisherValue">
    <property name="geometry">
     <rect>
      <x>837</x>
      <y>16</y>
      <width>147</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeriesValue">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>16</y>
      <width>167</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeries">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>16</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Серия:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>45</y>
      <width>968</width>
      <height>666</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>966</width>
       <height>664</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="actionBack"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Открыть папку с комиксами…</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Следующий файл</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Предыдущий файл</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
